### PR TITLE
Onchain account asset page

### PR DESCRIPTION
**Pull request by onnovisser**
### Description

Add asset page for onchain reserve with transaction table. use the same transaction table for cash assets as well.

#2014

### Approvals

- [ ] Dev

### Screenshots

image

image

### CODE REVIEW

File: centrifuge-app/src/components/PoolOverview/TransactionHistory.tsx [hunk 1]
```diff
@@ -82,7 +82,19 @@ export const columns = [
...
+export const TransactionHistoryTable = ({
+  transactions,
+  poolId,
+  preview = true,
+}: {
+  poolId: string
+  transactions: any[]
+  preview?: boolean
+}) => {
+  console.log('transactions', transactions)
```
Comment (sophialittlejohn) [inline centrifuge-app/src/components/PoolOverview/TransactionHistory.tsx:97]:
```suggestion

```

File: centrifuge-app/src/pages/Loan/index.tsx [hunk 2]
```diff
@@ -350,28 +376,38 @@ function Loan() {
...
-                <TransactionTable
-                  transactions={borrowerAssetTransactions}
-                  currency={pool.currency.symbol}
-                  loanType={
-                    'valuationMethod' in loan.pricing && loan.pricing.valuationMethod === 'oracle'
-                      ? 'external'
-                      : 'internal'
+              'valuationMethod' in loan.pricing && loan.pricing.valuationMethod === 'cash' ? (
+                <PageSection>
+                  <TransactionHistoryTable
```
Comment (sophialittlejohn) [inline centrifuge-app/src/pages/Loan/index.tsx:381]: hmm I think we should improve the naming of our components, I'm seeing a `<TransactionHistoryTable>` and a `<TransactionTable>` in the same file. Maybe they should be prefixed somehow so we know how they're different?

**Review by sophialittlejohn — APPROVED** [review]
Just 2 very minor comments, looks good otherwise 🔥

**Review by hieronx — APPROVED** [review]
Preview looks good to me!

File: centrifuge-app/src/pages/Loan/index.tsx (hunk 2 above)
Comment (onnovisser) [reply centrifuge-app/src/pages/Loan/index.tsx:382]: There's two now, but the use of old one will be replaced with the new one soon I believe

**Pull request merged**